packages/stream/solana/StreamClient.ts

```diff
@@ -199,13 +200,14 @@ export default class SolanaStreamClient extends BaseStreamClient {
       this.programId,
     );
 
-    const senderTokens = await ata(mintPublicKey, sender.publicKey);
-    const recipientTokens = await ata(mintPublicKey, recipientPublicKey);
-    const streamflowTreasuryTokens = await ata(mintPublicKey, STREAMFLOW_TREASURY_PUBLIC_KEY);
+    const { programId } = await getMintAndProgram(this.connection, mintPublicKey);
```

Comment:
It becomes harder to read because we have the `programId` here, instance member `programId`, `getProgramId` method in this class. Wdyt about renaming `getMintAndProgram` return value to tokenProgramId, for example?